bindings/aws/s3/metadata.yaml

```diff
@@ -0,0 +1,102 @@
+# yaml-language-server: $schema=../../../component-metadata-schema.json
+schemaVersion: v1
+type: bindings
+name: aws.s3
+version: v1
+status: stable
+title: "AWS S3"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-bindings/s3/
+binding:
+  output: true
+  operations:
+    - name: create
+      description: "Create blob"
+    - name: get
+      description: "Get blob"
+    - name: delete
+      description: "Delete blob"
+    - name: list
+      description: "List blob"
+capabilities: []
+builtinAuthenticationProfiles:
+  - name: "aws"
+metadata:
+  - name: bucket
+    required: true
+    description: |
+      The name of the S3 bucket to write to.
+    example: '"bucket"'
+    type: string
+  - name: region
+    required: true
+    description: |
+      The AWS region to the instance. See this page for valid regions:
```

Comment:
"AWS region to the instance" sounds odd. Maybe "The AWS region where the bucket exists"?